### ISSUE TITLE
Suggestion to obsolete White matter neuronal heterotopia

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#4507]
Related to https://github.com/obophenotype/human-phenotype-ontology/issues/4382
I will ask our COST:neuromig colleagues about this, since I see that this term has been used for Vici syndrome
https://www.ncbi.nlm.nih.gov/pubmed/10405446
 and for a few other diseases, e.g.: https://www.ncbi.nlm.nih.gov/pubmed/9413276

Renske, if we want to obsolete this term, we will have to find a replacement and merge/reannotate. Can you let us know what the prefered way of classifying this kind of heterotopia is?

**Comment by Renske-Oegema** [comment]
the neuronal heterotopia always consist of grey matter within the white matter, and the term grey matter heterotopia is more frequently used. However it is preffered to indicate more exact where the heterotopia are located  , eg periventricular or subcortical. 
heterotopic neurons within the white matter are a pathology finding, and indicate single neurons or small clusters of neuron. this is a rather frequent finding but not sufficient to call it heterotopia. 
the heterotopia in Vici syndrome are poorly described in this paper("an abnormal cortical structure with an area of white matter neuronal heterotopia") without an mri image.  this has not been described in other patients with vici syndrome. So this term is not specified anywhere.

**Comment by pnrobinson** [comment]
Hi Renske,
I would like to add the remaining terms. Could you please take a look at the issues like this?
Also, I would like to name this something like "Thick pachygyria" so that we do not have the detail of >10 mm in the term label. Would that be a good idea?
https://github.com/obophenotype/human-phenotype-ontology/issues/4513

thanks Peter

Peter Robinson
Professor and Donald A. Roux Chair, Genomics and Computational Biology
The Jackson Laboratory for Genomic Medicine
860.837.2095 t | peter.robinson@jax.org | https://robinsongroup.github.io/
Peter Robinson

________________________________

**Comment by pnrobinson** [comment]
I will merge this term.
